conda/meta.yaml

```diff
@@ -53,7 +53,7 @@ requirements:
         - pivy
         - pyside2
         - smesh
-        - netgen 6.2.1808*
+        - netgen 6.2.2203*
```

Comment:
@looooo are these conda files even used? I think everything is now on the freecad-feedstock repo isn't it?

Reply:
IMO better to remove it, if someone uses it they'll ask for it and we can direct them to the actually maintained files